ts-sdk/src/clients/modules/get.ts

```
@@ -65,7 +76,25 @@ export class Get {
+      // TODO: This keeps infinite loading state when
+      // returning the mocked block :/
+      return this.tendermintClient!.getBlock();
+
+      // Mock the Tendermint Block structure with Fuel data
+      // fuelBlock.time is already an ISO string from tai64ToDate().toISOString()
+      return {
+        header: {
+          height: fuelBlock.height,
+          time: fuelBlock.time,
+        },
+      } as Block;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)